src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+
+const numbers = Array.from(populations).map(population =>
```

Comment:
Consider using a more descriptive variable name instead of 'numbers'. For example, 'populationNumbers' would be more informative.